### PR TITLE
oVirt: general improvements

### PR DESCRIPTION
**Pull request by dougsland**
- ovirt: add checkUrlResponse() for credentials
- ovirt: add PEM URL to config struct
- ovirt: yaml file stored in ${HOME}/.config/ovirt/
- ovirt: Ask users FQDN instead of API url
- ovirt: additional improvements
       - Auto-detect if users already have locally imported CA from engine
      - In case the CA Cert is not locally imported, offer to user to
        download and import automatically
      - Remove the need of copy/past certificate, auto download and install
        cert (if user allow)
      - Replace oVirt string to Engine as it is common name in upstream and
        downstream

### CODE REVIEW

**Comment by dougsland** [comment]
@rgolangh @gekorob @Gal-Zaidman please review.

File: pkg/asset/installconfig/ovirt/credentials.go [hunk 1]
```diff
@@ -2,12 +2,25 @@ package ovirt
...
 import (
 	"fmt"
+	"net/http"
 	"net/url"
 
 	"github.com/sirupsen/logrus"
 	"gopkg.in/AlecAivazis/survey.v1"
 )
 
+// Check if URL can be reacheable before we proceed with the installation
```
Comment (sandrobonazzola) [inline pkg/asset/installconfig/ovirt/credentials.go:12]: "can be reachable" -> "can be reached" or "is reachable"

**Comment by dougsland** [comment]
Issue not related to the patch:

W0521 21:09:18.535999       1 loader.go:223] Config not found: /var/run/secrets/ci.openshift.io/multi-stage/kubeconfig
error: failed to load cluster config: could not load client configuration: invalid configuration: no configuration has been provided

/retest

**Comment by dougsland** [comment]
timeout ci

**Comment by dougsland** [comment]
2020/05/24 10:12:32 No custom metadata found and prow metadata already exists. Not updating the metadata.
2020/05/24 10:12:33 Ran for 18s
error: some steps failed:
  * could not run steps: step upi-installer failed: could not wait for build: the build upi-installer failed with reason DockerBuildFailed: Docker build strategy has failed.
0      0      0 --:--:--  0:01:46 --:--:--     0
  0     0...i.vmware.devcluster.openshift.com:443; Connection timed out
error: build error: running 'curl -L -O -k https://vcsa-ci...ors &&     update-ca-trust extract' failed with exit code 7
2020/05/24 10:12:33 could not load result reporting options: open : no such file or directory

**Comment by rgolangh** [comment]
This PR has too much going around. Need to split this to logical PRs.

Also see my comment on https://bugzilla.redhat.com/show_bug.cgi?id=1838695#c1 for the config folder. I don't think this is needed. And definitely a change like that would be suprizing for existing users.

**Comment by dougsland** [comment]
Going to open a different pull request.

**Pull request closed without merging**